source/_docs/email.md

```diff
@@ -35,14 +70,60 @@ While many providers have standalone modules/plugins, you may want or need to in
 
     <div class="alert alert-info" role="alert">
       <h4 class="info">Note</h4>
-        <p>Support for Drupal 8 is not yet available for the <a href="https://www.drupal.org/project/smtp"> SMTP Authentication Support</a> module. However, <a href="https://groups.google.com/a/pantheon.io/forum/#!topic/power-users/HxvK7T0MPEM"> some users</a> have reported success with the pre-release version. For details, see <a href="/docs/guides/sendgrid/#sendgrid-smtp-integration"> this article</a>.</p>
+        <p>Support for Drupal 8 is in Beta. <a href="https://www.drupal.org/project/smtp"> SMTP Authentication Support</a> module.</p>
     </div>
   </div>
   <div role="tabpanel" class="tab-pane" id="wp">
     Once you have chosen your SMTP provider, install and configure Wordpress's <a href="https://wordpress.org/plugins/wp-mail-smtp/">WP Mail SMTP</a> plugin.
   </div>
 </div>
 
+Customers have successfully used [SendGrid](/docs/guides/sendgrid/), Gmail, Amazon SES, Mandrill, or other externally hosted SMTP based email providers.
+
+Since we block traffic on ports 25, 465 and 587, here’s a list of popular email providers and the additional ports which will work on Pantheon. We suggest you refer to the corresponding documentation for any updated information.
+
+<table class="table table-condensed table-bordered">
+    <thead class="thead-inverse">
+      <tr>
+        <th scope="row" class="thead-inverse">Provider</th>
+        <th>Port Documentation</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row" class="thead-inverse">SendGrid</th>
+        <td><a href="https://sendgrid.com/docs/Classroom/Basics/Email_Infrastructure/smtp_ports.html" target="_blank">2525</a></td>
+      </tr>
+      <tr>
+        <th scope="row" class="thead-inverse">Mandrill</th>
+        <td><a href="https://mandrill.zendesk.com/hc/en-us/articles/205582167-Which-SMTP-ports-can-I-use-" target="_blank">2525</a></td>
+      </tr>
+      <tr>
+        <th scope="row" class="thead-inverse">Mailgun</th>
+        <td><a href="http://blog.mailgun.com/25-465-587-what-port-should-i-use/" target="_blank">2525</a></td>
+      </tr>
+      <tr>
+        <th scope="row" class="thead-inverse">SparkPost</th>
+        <td><a href="https://www.sparkpost.com/docs/faq/smtp-connection-problems/" target="_blank">2525</a></td>
+      </tr>
+      <tr>
+        <th scope="row" class="thead-inverse">Amazon SES</th>
+        <td><a href="http://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-connect.html" target="_blank">2587 (STARTTS), 2465 (TLSWRAPPER)</a></td>
+      </tr>
+    </tbody>
+</table>
+
+If you do not find your service provider in the table above, kindly check with their support/ documentation.
+
+#### Approach 3: Pantheon via Local MTA (may result in emails not being delivered)
```

Comment:
@rachelwhitton I wonder if we remove it, whether CSE will have an overhead of answering a question like "Do you not at all support local MTA?"/ "What if, I don't want to use external provider - does that mean I can send no emails?" .. stuff like that. And then they will have to maintain another set of response somewhere.

I think I can work on pulling out from that list and putting it into a separate bucket to make it appear as unsupported feature. Thoughts?